src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/data/Prison.kt

```diff
@@ -1,7 +1,3 @@
 package uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data
 
-class Prison(id: String, name: String, active: Boolean) {
-  val id: String = id
-  val name: String = name.replace(" \\(.*\\)".toRegex(), "")
-  val active: Boolean = active
-}
+data class Prison(val id: String, val name: String, val active: Boolean)
```

Comment:
Making this a data class makes testing a lot easier and we don't need the custom name logic anymore as we are using caseloads now.